### PR TITLE
Add GitReleaseNotes alias

### PR DESCRIPTION
**Pull request by gep13**
@devlead @patriksvensson would love it if you could review this.

Thanks!

### CODE REVIEW

**Comment by gep13** [comment]
@jakeginnivan would you be in a position to take a look at this as well?  Would love your input, thanks!

**Comment by gep13** [comment]
I am just looking again at this, and I think I need to change the GitReleaseNotes Alias.  Looking at this:

https://github.com/GitTools/GitReleaseNotes/blob/master/src/GitReleaseNotes/Program.cs#L35

It would appear that the OutputFile is a mandatory command line parameter, which makes sense, so I will refactor the Alias to account for that.

**Comment by devlead** [comment]
@gep13 Agree mandatory arguments should be parameters.

**Comment by gep13** [comment]
:+1:

File: src/Cake.Common/Tools/GitReleaseNotes/IssueTracker.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿namespace Cake.Common.Tools.GitReleaseNotes
+{
+    /// <summary>
+    /// The IssueTracker to be used.
+    /// </summary>
+    public enum IssueTracker
```
Comment (devlead) [inline src/Cake.Common/Tools/GitReleaseNotes/IssueTracker.cs]: @gep13 as this basically becomes a global available enum we should probably prefix if with GitReleaseNotes.

Comment (gep13) [inline src/Cake.Common/Tools/GitReleaseNotes/IssueTracker.cs]: As in `GitReleaseNotesIssueTracker`?

Comment (devlead) [inline src/Cake.Common/Tools/GitReleaseNotes/IssueTracker.cs]: Yes. All alias's namespaces  are added when imported, names are basically global and currently this usages is pretty "local".

Comment (gep13) [inline src/Cake.Common/Tools/GitReleaseNotes/IssueTracker.cs]: Gotcha, that makes sense, I will fix this up.

**Comment by gep13** [comment]
@devlead I have fixed up the issues that you mentioned in the earlier review of this, any chance you can take another look, and let me know what you think?  Cheers!

**Comment by gep13** [comment]
@devlead looks like the Travis build failed due to timeout, can you kick it again?

**Comment by patriksvensson** [comment]
@gep13 Kicked!

**Comment by gep13** [comment]
@devlead @patriksvensson I "think" I have refactored this tool as well, would appreciate any feedback, thanks!

File: src/Cake.Common/Tools/GitReleaseNotes/GitReleaseNotesAliases.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+
+namespace Cake.Common.Tools.GitReleaseNotes
+{
+    using global::Cake.Core.IO;
```
Comment (patriksvensson) [inline src/Cake.Common/Tools/GitReleaseNotes/GitReleaseNotesAliases.cs]: Is there a reason this is using the global alias (and inside the namespace)?

**Comment by patriksvensson** [comment]
@gep13 Left a comment. Other than that this looks good to me.

File: src/Cake.Common/Tools/GitReleaseNotes/GitReleaseNotesAliases.cs (hunk 2 above)
Comment (gep13) [inline src/Cake.Common/Tools/GitReleaseNotes/GitReleaseNotesAliases.cs]: Yes :-( ReSharper is currently fighting me, and I haven't sat down and looked into it yet.  I have ReSharper set up with all the default rules, including putting usings with the namespaces, which isn't what cake does.  As a result, each time I have to remember to put them back properly.  I am starting to move away from that rule in general though, and I need to turn off that refactoring within ReSharper, I just need to find where it is :smile: 

I also found another instance of `global::` within one of the files for GitReleaseManager, and although not related to this PR, I took the opportunity to change it just now in this PR.  Let me know if you want me to back that out.

**Comment by gep13** [comment]
@patriksvensson updated and rebased.

**Comment by devlead** [comment]
@gep13 merged, thanks :+1:

**Pull request merged**